patches/ios-web-web_state-ui-wk_web_view_configuration_provider.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/ios/web/web_state/ui/wk_web_view_configuration_provider.mm b/ios/web/web_state/ui/wk_web_view_configuration_provider.mm
+index a3e5cdad9454cda0a49986b2a67f1b11741ad7d3..538b2a93c65fc3912a2fb5cabde42ef640e9c9b0 100644
+--- a/ios/web/web_state/ui/wk_web_view_configuration_provider.mm
++++ b/ios/web/web_state/ui/wk_web_view_configuration_provider.mm
+@@ -143,6 +143,7 @@ void WKWebViewConfigurationProvider::ResetWithWebViewConfiguration(
+       fetchDataRecordsOfTypes:data_types
+             completionHandler:^(NSArray<WKWebsiteDataRecord*>* records){
+             }];
++  BRAVE_RESET_WITH_WEB_VIEW_CONFIGURATION
```

Comment:
I thought we were going to subclass for this?

Reply:
No the goal was to use the observer for this but it turned out there's no way to add the observer early enough to catch the first configuration creation. I can see if its possible to somehow subclass this though

Reply:
fixed in https://github.com/brave/brave-core/pull/26957/commits/de45e48ba436e00fffcead74f84106b87874bc44